2021-offseason/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +14,21 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class kDrivetrain{
+        public static final int frontLeft_ID = 18;
+        public static final int frontRight_ID = 15;
+        public static final int backLeft_ID = 1;
+        public static final int backRight_ID = 14;
```

Comment:
For all final values in java you should name your variables upper case with underscores in between each word
`public static final int FRONT_LEFT_ID`